apps/web/components/dispute/DisputeList.tsx

```
@@ -0,0 +1,112 @@
+                                            fontWeight: 600,
+                                            fontSize: 13
+                                        }}>
+                                            {d.approved === null ? 'Pending' : d.approved ? 'Approved' : 'Rejected'}
+                                        </span>
+                                    </td>
+                                    <td style={{ padding: 12 }}>{d.review_notes}</td>
+                                    <td style={{ padding: 12 }}>
+                                        <button
+                                            style={{ background: '#4a90e2', color: '#fff', border: 'none', padding: '6px 14px', borderRadius: 6, fontWeight: 600, cursor: 'pointer' }}
+                                            onClick={() => setExpanded(expanded === d.id ? null : d.id)}
+                                        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)